lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
Great!  You found it, but {2048,17} still fails. However {2048,65537} works!
And don't forget the Valgrind fix that Sverker commented on.

Just for your information: when I test this in the shell, with fips disabled it takes 0.2-0.3 s to generate the key pair, but with fips enabled 0.9-1.3 s!  So the fips mode does more than simply checking the sizes.